package.json

```diff
@@ -22,14 +22,20 @@
     "react-autocomplete": "^1.8.1",
     "react-chartjs2": "^1.2.1",
     "react-dom": "^16.13.1",
-    "react-dropzone": "^8.1.0",
+    "react-dropzone": "4.3.0",
     "react-markdown": "^4.3.1",
     "react-scroll": "^1.7.12",
     "react-spring": "^8.0.27",
     "reactstrap": "^8.4.1",
     "sjtest": "^0.4.9",
     "wwutils": "^0.9.5",
-    "you-again": "^0.8.8"
+    "you-again": "^0.8.8",
+    "jest": "^25.1.0",
```

Comment:
Hello Ben

jest, shelljs, puppeteer, puppet-theatre, yargs -- these look like devDependencies, rather than dependencies.

If they're only used in build or test -- I think they go in devDependencies.
If they're used in the web-app itself, they go in dependencies

Also -- Please aim for alphabetical order. We don't consistently do that, but it's a good idea -- makes comparisons easier.

Thanks,
 - Daniel

Reply:
I didn't add those myself, so they must have been added in automatically by npm during an `npm install` or something similar. I'll rearrange them!